### ISSUE TITLE
User testing: IPFS Colorado in-person test day

### ISSUE THREAD

**Issue by jessicaschilling** [ipfs/docs#355]
# User Testing Summary

## Event
**[IPFS Colorado User Testing Day](https://meetup.com/IPFS-Colorado/events/264964856/)**
Boulder, Colorado, 16 Oct 2019

In conjunction with Ethereum Boulder, we co-hosted a free full-day event for anyone interested in or already experienced with IPFS and the decentralized web. Participants during the daytime event were offered a day of free coworking at [The Peregrine](https://theperegrine.co), plus some delightful IPFS swag, in exchange for a half-hour, in-person interview.

Evening participants were treated to an "IPFS Basics" interactive presentation from ProtoSchool chapter organizer and community dude extraordinaire @nukemandan. Plus more fun IPFS swag! You can [view the presentation here](https://www.youtube.com/watch?v=D3MjB45YZsM).

## Researchers
@ericronne and @jessicaschilling  

## Methods
Participants RSVP'd ahead of time, but it wasn't possible to discern clearly what their existing level of interest in or involvement with IPFS was; keeping this in mind, we opted to keep testing extremely open-ended and conversational, leading with some initial interest/skill inventory questions and from there discussing one or more artifacts depending on their response: an outline version of our proposed beta site navigation structure (worked in issue #273), the "no content here yet" page proposed for our beta site (worked in issue #342), and/or our goal-based IPFS user personae (see [Mural doc](https://app.mural.co/t/protocollabs6957/m/protocollabs6957/1564779785852/cf7669f3c1773508a811a3fa0eadfb99efb310bf)).

We had **eight participants** with a widely varying degree of existing expertise, a broad variety of reasons for caring about IPFS and the dweb, and a very wide-ranging spectrum of goals and end-game interests. Identifying characteristics have been removed in our interview notes to protect privacy.

## Take-aways

**[Please view the full test summary here for all the take-aways!](https://hackmd.io/WU9USzgGQmSEyWusy9WMbg?view)**

TLDR: This exercise was primarily a level-setting endeavor; we were hoping to confirm/deny overall validity of our foundational efforts in Q3 and the beginning of Q4 2019 before going full-tilt into building out the new docs beta site. While we definitely received insights that changed some details of the build, it was encouraging to see that there weren't any significant holes in our approach so far. This test also provides us with a useful baseline for comparison for future user tests on the docs beta in Q4 -- as well as some wonderful Colorado connections for future rounds of testing!

## Test day artifacts

- [Workflowy of beta nav structure](https://workflowy.com/s/ipfs-docs/odHb7KUgJZyae6bi)
- [Flat mockup of "nothing here yet" page](https://user-images.githubusercontent.com/9259185/66786749-289f7a00-eea7-11e9-90bb-3a3237182677.png)
- [Persona overview](https://app.mural.co/t/protocollabs6957/m/protocollabs6957/1564779785852/cf7669f3c1773508a811a3fa0eadfb99efb310bf)
- [Raw notes from user interviews](https://hackmd.io/-XGjBQA7R021IOSQFUMUwQ?view)

**Comment by jessicaschilling** [comment]
- Edited this issue's original comment to include a research summary and link to @ericronne's excellent notes
- Referenced the summary in epic issue #288 (so we can use that epic issue as a summary of our research work for the quarter) and linked to the epic in our repo readme (so it's easy for the community to see what we've been researching and testing)
- Added some action items and their relevant links to Eric's notes document, so we don't lose track of acting on what we learned

@ericronne -- please feel free to edit the issue's original comment if you want to change/append anything. Otherwise, I believe we've captured enough actionable results of our day in Boulder to consider this issue closed. Thanks to all!
